### DISCUSSION TITLE
Vanilla Icons

### DISCUSSION THREAD

**Discussion by emwedk** [witalihirsch/qBitTorrent-fluent-theme#42] [Ideas]
@witalihirsch maybe add vanillaIcons theme files for people that rather wanna use qbits original icons? literally takes a second for us :)
since you changed some icons for your liking, you might aswell do it from your end

**Comment by witalihirsch** [comment]
We can make two folders. First one for fluent icons and second for default icons :)

**Reply by emwedk** [reply]
good call !

**Comment by emwedk** [comment]
btw, here's the source for the icon names https://github.com/qbittorrent/qBittorrent/tree/v4_5_x/src/icons

**Reply by witalihirsch** [reply]
it'll be useful, thanks!
